app/assets/javascripts/cookie_functions.js

```diff
@@ -0,0 +1,26 @@
+window.TeacherPayments = window.TeacherPayments || {};
+
+window.TeacherPayments.cookies = {
+  acceptCookieName: "accept_cookies",
+  checkNonEssentialCookiesAccepted: function() {
+    return this._get(this.acceptCookieName);
+  },
+  acceptNonEssentialCookies: function() {
+    this._set(this.acceptCookieName, 1, 90);
+  },
+  setNonEssentialCookie: function(name, value, days) {
```

Comment:
No, I put a note about it in the commit message. I wanted to make it clear in code that if you want to set a non-essential cookie the user needs to give consent. Unfortunately the only non-essential cookies we have are set by Google Analytics. Happy to remove it though if you think we don't need it

Reply:
I think if we want to communicate that there's probably a more explicit and clear way to do it. the chances of a developer that needs to add a non-essential cookie coming across this method and understanding that they need to call it is pretty slim. Perhaps it would be better to simply document it somewhere?

Reply:
I've removed this unused function and added a note to the readme about storing non-essential cookies